### PR TITLE
`safearea.get_insets()` for android

### PR DESCRIPTION
**Pull request by AGulev**

### CODE REVIEW

File: game.project [hunk 1]
```diff
@@ -26,7 +26,7 @@ launch_image_1125x2436 = /main/launch_images/ios_launch_1125x2436.png
 launch_image_2436x1125 = /main/launch_images/ios_launch_2436x1125.png
 
 [android]
-package = com.defold.examples.iPhoneXSafearea
+package = com.defold.examples.Safearea
```
Comment (britzl) [inline game.project:29]: Let's change this to lowercase letters!

File: game.project [hunk 2]
```diff
@@ -5,10 +5,10 @@ main_collection = /main/main.collectionc
 shared_state = 1
 
 [project]
-title = iPhoneXSafearea
+title = Safearea
 
 [ios]
-bundle_identifier = com.defold.examples.iPhoneXSafearea
+bundle_identifier = com.defold.examples.Safearea
```
Comment (britzl) [inline game.project:11]: Let's change this to lowercase letters!

**Review by britzl — CHANGES_REQUESTED** [review]
We also need to update safearea_null.cpp to take into account Android

File: safearea/src/safearea_null.cpp [hunk 3]
```diff
@@ -1,11 +1,11 @@
 #include "safearea.h"
 
-#ifndef DM_PLATFORM_IOS
+#ifndef DM_PLATFORM_IOS && DM_PLATFORM_ANDROID
```
Comment (britzl) [inline safearea/src/safearea_null.cpp:3]: Is this working? I've learned that for multiple define checks you need to use defined() like this::

`#if !defined(DM_PLATFORM_IOS) && !defined(DM_PLATFORM_ANDROID)`

Comment (AGulev) [reply safearea/src/safearea_null.cpp]: Good point, thank you!

**Review by britzl — APPROVED** [review]
Looks good!

**Pull request merged**